pkg/hvs/postgres/flavortemplate_store.go

```diff
@@ -163,3 +171,36 @@ func (ft *FlavorTemplateStore) Recover(recoverTemplates []string) error {
 
 	return nil
 }
+
+func (ft *FlavorTemplateStore) buildFlavorTemplateSearchQuery(tx *gorm.DB, criteria *models.FlavorTemplateFilterCriteria) *gorm.DB {
+	defaultLog.Trace("postgres/flavortemplate_store:buildFlavorTemplateSearchQuery() Entering")
+	defer defaultLog.Trace("postgres/flavortemplate_store:buildFlavorTemplateSearchQuery() Leaving")
+
+	if tx == nil {
+		return nil
+	}
+
+	tx = tx.Model(&flavorTemplate{})
+	if criteria == nil {
+		return tx
+	}
+
+	if !criteria.IncludeDeleted {
+		tx = tx.Where("deleted = ?", false)
+	}
+
+	if criteria.Id != uuid.Nil {
+		tx = tx.Where("id = ?", criteria.Id)
+	}
+	if criteria.Label != "" {
+		tx = tx.Where(convertToPgJsonqueryString("content", "label")+" = ?", criteria.Label)
+	}
+	if criteria.ConditionContains != "" {
+		tx = tx.Where(convertToPgJsonqueryString("content", "condition")+" like ?", "%"+criteria.ConditionContains+"%")
+	}
+	if criteria.FlavorPartContains != "" {
+		tx = tx.Where(convertToPgJsonqueryString("content", "flavor_parts")+" like ?", "%"+criteria.FlavorPartContains+"%")
```

Comment:
Where "convertToPgJsonqueryString" is defined?

Reply:
"convertToPgJsonqueryString" method is defined in intel-secl/pkg/hvs/postgres/flavor_store.go